web_boolean_button/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Boolean Button Widget",
+    "version": "13.0.0.1.0",
+    "category": "Web",
+    "summary": "",
+    "license": "LGPL-3",
```

Comment:
If I'm correct, this should be [OPL](https://www.odoo.com/documentation/user/14.0/legal/licenses/licenses.html#odoo-apps) instead, since it's a forward port of code from the 12.0 `web` module. @pedrobaeza Can I tap your knowledge on this one?

Reply:
No, we can't publish anything OPL here in OCA. It should respect the previous license, so I think `LGPL-3` seems correct if this comes from Odoo core v12. Isn't this something of enterprise, right?

You should put anyway the disclaimer about being Odoo initial code, and also you need to make a first commit attributing the original code to "Odoo S. A. <info@odoo.com>", and then put successive commits from your part with the needed changes to adapt to the version. See for example https://github.com/OCA/manufacture/pull/501

Reply:
No, it's CE 12.0. Thanks for the clarification!